modules/core/src/experimental/utils/gpu-grid-aggregation/gpu-grid-aggregator.js

```diff
@@ -19,6 +20,40 @@ const DEFAULT_CHANGE_FLAGS = {
   viewportChanged: true,
   cellSizeChanged: true
 };
+const DEFAULT_RUN_PARAMS = {
```

Comment:
Any way you can break down this monster file (constants, utils)?

Reply:
Done.